demo/index.html

```diff
@@ -68,6 +70,10 @@
       padding: 8px;
     }
 
+    .flex {
```

Comment:
In the toolbar demo I actually called this `spacer`, which kind of makes more sense, imho, and makes it look less like you're just copy pasting the flex class.